src/vue/modules/withdrawal-fiat-bank/index.vue

```diff
@@ -0,0 +1,389 @@
+        v-model.trim="form.bankName"
+        name="withdrawal-fiat-bank-bank-name"
+        :label="'withdrawal-fiat-bank-module.bank-name' | globalize"
+        :disabled="formMixin.isDisabled"
+        @blur="touchField('form.bankName')"
+        :error-message="getFieldErrorMessage('form.bankName')"
+      />
+    </div>
+    <div class="app__form-row">
+      <input-field
+        white-autofill
+        class="app__form-field"
+        v-model.trim="form.iban"
+        name="withdrawal-fiat-bank-iban"
+        :label="'withdrawal-fiat-bank-module.iban' | globalize"
+        :disabled="formMixin.isDisabled"
+        @blur="touchField('form.iban')"
+        :error-message="getFieldErrorMessage('form.iban')"
+      />
+    </div>
+    <div class="app__form-actions withdrawal__action">
+      <button
+        v-ripple
+        v-if="!formMixin.isConfirmationShown"
+        type="submit"
+        class="app__button-raised"
+        :disabled="formMixin.isDisabled"
+        form="withdrawal-fiat-bank-module"
+      >
+        {{ 'withdrawal-fiat-bank-module.withdrawal' | globalize }}
+      </button>
+      <form-confirmation
+        v-if="formMixin.isConfirmationShown"
+        :is-pending="isWithdrawalPending"
+        @ok="submit"
+        @cancel="hideConfirmation()"
+      />
+    </div>
+  </form>
+  <loader v-else message-id="withdrawal-fiat-bank-module.loading-msg" />
+</template>
+
+<script>
+import Loader from '@/vue/common/Loader'
+
+import { mapActions, mapMutations, mapGetters } from 'vuex'
+import FormMixin from '@/vue/mixins/form.mixin'
+import debounce from 'lodash/debounce'
+import { ErrorHandler } from '@/js/helpers/error-handler'
+import { types } from './store/types'
+import { Bus } from '@/js/helpers/event-bus'
+import { Wallet, base } from '@tokend/js-sdk'
+import { initApi, api } from './_api'
+import {
+  required,
+  noMoreThanAvailableOnBalance,
+  maxDecimalDigitsCount,
+  maxLength,
+  ibanValidator,
+} from '@validators'
+
+const EMPTY_FEE = '0.0000'
+
+const EVENTS = {
+  withdrawn: 'withdrawn',
+}
+
+export default {
+  name: 'withdrawal-fiat-bank-module',
+  components: {
+    Loader,
+  },
+  mixins: [FormMixin],
+  props: {
+    wallet: {
+      type: Wallet,
+      required: true,
+    },
+    /**
+     * @property config - the config for component to use
+     * @property config.horizonURL - the url of horizon server (without version)
+     * @property config.decimalPoints - count of allowed decimal points
+     * @property config.minAmount - minimal allowed amount
+     */
+    config: {
+      type: Object,
+      required: true,
+    },
+  },
+  data: _ => ({
+    isInitialized: false,
+    isFeesLoadPending: false,
+    isFeesLoadFailed: false,
+    isWithdrawalPending: false,
+    feesDebouncedRequest: null,
+    fees: {
+      fixedFee: EMPTY_FEE,
+      percentFee: EMPTY_FEE,
+    },
+    form: {
+      asset: null,
+      amount: null,
+      userName: null,
+      bankName: null,
+      iban: null,
```

Comment:
I believe BIC/SWIFT code field is missing?

Reply:
Used all fields that Aleksandr Plotnikov send. Do we need to add a `BIC/SWIFT ` too?

Reply:
@letehaha yep